### PR TITLE
LPS-166264 Add tests to auto deploy zip files through batch engine

### PR DESCRIPTION
**Pull request by yanan**
**Background:**
- Add tests for new story: Enable auto-deploy of multiple entities in one zip file

**Test Plan:**
- Given a BlogPosting.zip file with siteId
When I add it to deploy folder
Then all blog postings are imported
- Given a content structure is created
Given a Structured.zip file with content structure id
When I add it to deploy folder
Then all structured contents are imported
- Given an ObjectDefinition.zip file
When I add it to deploy folder
Then all object definitions are imported
- Given a MultipleBatchPairs.zip file including blog with valid siteId and object definition
When I add it to deploy folder
Then all elements included in the file are imported

**Jira Ticket:**
- https://issues.liferay.com/browse/LPS-166264

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
Test failures are unrelated.

**Comment by yanan** [comment]
ci:test:sf

File: modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/objectDefinition/export.jsonl [hunk 1]
```diff
@@ -0,0 +1,2 @@
...objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"Date"},{"DBType":"String","businessType":"Text","defaultValue":"","externalReferenceCode":"dfe16957-e219-fd42-c754-4f6188403e94","id":43708,"indexed":true,"indexedAsKeyword":false,"indexedLanguageId":"en_US","label":{"en_US":"name"},"listTypeDefinitionExternalReferenceCode":"","name":"name","objectFieldSettings":[{"name":"showCounter","value":"false"}],"required":false,"state":false,"system":false,"type":"String"},{"DBType":"String","businessType":"Text","defaultValue":"","externalReferenceCode":"0696b345-c2c2-e305-061c-e6f7d32ad3df","id":43692,"indexed":false,"indexedAsKeyword":false,"indexedLanguageId":"","label":{"en_US":"Status"},"listTypeDefinitionExternalReferenceCode":"","name":"status","objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"String"}],"objectLayouts":[],"objectRelationships":[],"objectViews":[],"panelCategoryKey":"","parameterRequired":false,"pluralLabel":{"en_US":"TestObjects"},"portlet":false,"restContextPath":"/o/c/testobjects","scope":"company","status":{"code":0,"label":"approved","label_i18n":"Approved"},"system":false,"titleObjectFieldName":"createDate"}
```
Comment (magjed4289) [inline modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/objectDefinition/export.jsonl:1]: to me it looks okay, but maybe Brian will ask us to make it more readable 🤔 Not sure here 😅

**Review by magjed4289 — APPROVED** [review]
looks good, some small changes only! and one doubt about jsonl file formatting :D

**Review by magjed4289 — COMMENTED** [review]
leaving some comments :)

File: modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/multipleElmentsParis/batch-engine.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
[the remark this answers is not in the document]
Comment (yanan) [reply modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/multipleElmentsParis/batch-engine.json]: Great, thanks for catching this. I'll update before forward. :heart:

File: modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/objectDefinition/export.jsonl (hunk 1 above)
Comment (yanan) [reply modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/objectDefinition/export.jsonl]: That's right, maybe I can sf manually first :smile:

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Review by Hijky — APPROVED** [review]

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:forward

**Comment by Hijky** [comment]
ci:forward

**Comment by Hijky** [comment]
The failures are not related to the PR, we can forward it manually.

**Comment by magjed4289** [comment]
[forwarded manually to Brian](https://github.com/brianchandotcom/liferay-portal/pull/128026)

**Pull request closed without merging**